app/src/main/res/layout-v22/myproject_setting.xml

```diff
@@ -43,13 +43,13 @@
                     <LinearLayout android:gravity="center_vertical" android:orientation="horizontal" android:paddingLeft="4.0dip" android:paddingRight="4.0dip" android:layout_width="fill_parent" android:layout_height="wrap_content" android:layout_marginTop="8.0dip">
                         <ImageView android:layout_width="24.0dip" android:layout_height="24.0dip" android:layout_marginRight="4.0dip" android:src="@drawable/code_96" android:scaleType="fitCenter" android:contentDescription="@null" />
                         <LinearLayout android:orientation="horizontal" android:background="@drawable/bg_outline" android:layout_width="0.0dip" android:layout_height="wrap_content" android:layout_marginLeft="8.0dip" android:layout_weight="1.0">
-                            <TextView android:textSize="14.0sp" android:textColor="#ff757575" android:gravity="center" android:id="@id/ver_code" android:background="?android:selectableItemBackground" android:layout_width="0.0dip" android:layout_height="fill_parent" android:text="1" android:layout_weight="1.0" />
+                            <TextView android:textSize="14.0sp" android:textColor="#ff757575" android:gravity="center" android:id="@id/ver_code" android:background="?attr/selectableItemBackground" android:layout_width="0.0dip" android:layout_height="fill_parent" android:text="1" android:layout_weight="1.0" />
```

Comment:
Where did you find out this was deprecated? The [current official docs of `android.R.attr`](https://developer.android.com/reference/android/R.attr#selectableItemBackground) don't tell to not use it.

Reply:
Check this here-
https://stackoverflow.com/questions/19714682/android-apply-selectableitembackground-in-xml-with-support-v7